### ISSUE TITLE
Attribute 'outDir' does not exist

### ISSUE THREAD

**Issue by cblp** [hoovercj/vscode-ghc-mod#24]
I simply do what README says and get:

```
Attribute 'outDir' does not exist ('/home/cblp/dev/vscode-ghc-mod/server/../client/server').: Error: Attribute 'outDir' does not exist ('/home/cblp/dev/vscode-ghc-mod/server/../client/server').
    at Object.C [as create] (file:////usr/share/code/resources/app/out/vs/workbench/workbench.main.js:34:19570)
    at file:////usr/share/code/resources/app/out/vs/workbench/workbench.main.js:105:21677
    at Object.m [as _notify] (file:////usr/share/code/resources/app/out/vs/workbench/workbench.main.js:30:8153)
    at Object.B.enter (file:////usr/share/code/resources/app/out/vs/workbench/workbench.main.js:30:11658)
    at i.Class.define._run (file:////usr/share/code/resources/app/out/vs/workbench/workbench.main.js:30:12979)
    at i.Class.define._chainedError (file:////usr/share/code/resources/app/out/vs/workbench/workbench.main.js:30:12338)
    at i (file:////usr/share/code/resources/app/out/vs/workbench/workbench.main.js:30:10161)
    at i.Class.define.then (file:////usr/share/code/resources/app/out/vs/workbench/workbench.main.js:30:13788)
    at Object.P.enter (file:////usr/share/code/resources/app/out/vs/workbench/workbench.main.js:30:10230)
    at i.Class.define._run (file:////usr/share/code/resources/app/out/vs/workbench/workbench.main.js:30:12979)
```

**Comment by hoovercj** [comment]
"I simply do what README says and get:"

You are doing what the README says to try and build the extension from source?

I think you need to compile the server directory first which will create the server directory in the outdir.

I'm at work so I'll take a look at this tonight and update the README with clearer instructions

**Comment by cblp** [comment]
README doesn't mention building, it only says `npm install`, and I did it.

Anyway, I already fixed it in #26.

**Comment by hoovercj** [comment]
I have updated the README to include instructions about building. Thanks for pointing out a place for improvement!
